.eslintrc.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ["@homebound"],
```

Comment:
Hmm, I actually think the homebound eslint config is private. That would make this repo challenging to work with for OSS contributors. Maybe I should use a standard public config instead?

Reply:
If we don't care about leaking internals like the `no-b-add-column` thing then I don't see why it couldn't be public.